src/main/java/com/google/api/codegen/discovery/MainDiscoveryProviderFactory.java

```diff
@@ -54,6 +59,16 @@
 
   private static final String DEFAULT_SNIPPET_FILE = "discovery_fragment.snip";
 
+  private static final Map<String, Class<? extends SampleMethodToViewTransformer>>
+      SAMPLE_METHOD_TO_VIEW_TRANSFORMER_MAP =
+          ImmutableMap.of(
+              JAVA,
+              JavaSampleMethodToViewTransformer.class,
+              NODEJS,
+              NodeJSSampleMethodToViewTransformer.class);
+  private static final Map<String, Class<? extends TypeNameGenerator>> TYPE_NAME_GENERATOR_MAP =
+      ImmutableMap.of(JAVA, JavaTypeNameGenerator.class, NODEJS, NodeJSTypeNameGenerator.class);
```

Comment:
I'll see if I can make it happen, this is something that the formatter seems to enforce... It'll probably be the case that once another language is added, the line will be long enough that the formatter will break it into the way you want.